packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -1922,32 +1943,21 @@ describe('Checkout', () => {
         describe('onFormSubmit ::', () => {
             let isFormValidSpy;
             let submitCheckoutSpy;
+            let trackFormInteractionSpy;
 
             beforeEach(() => {
                 isFormValidSpy = jest.spyOn(VueCheckout.methods, 'isFormValid');
                 submitCheckoutSpy = jest.spyOn(VueCheckout.methods, 'submitCheckout');
+                trackFormInteractionSpy = jest.spyOn(VueCheckout.methods, 'trackFormInteraction');
             });
 
             afterEach(() => {
                 jest.clearAllMocks();
             });
 
-            it('should emit `CheckoutValidationError` with validation state if form is invalid', async () => {
+            it('should call `trackFormInteractionSpy` with correct `action`', async () => {
                 // Arrange
-                const mockValidationState = {
```

Comment:
The test hasn't been deleted, I've moved it into `when the form is Invalid` describe block.